minimesos/src/main/java/com/containersol/minimesos/container/AbstractContainer.java

```diff
@@ -52,6 +52,10 @@ public AbstractContainer(MesosCluster cluster, String uuid, String containerId,
         this.config = config;
     }
 
+    private int getSystemPropertyLength(String property) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Make "getSystemPropertyLength" a "static" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://ci.infra.container-solutions.com/sonar/coding_rules#rule_key=squid%3AS2325)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Private method 'getSystemPropertyLength' is never used. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://ci.infra.container-solutions.com/sonar/coding_rules#rule_key=squid%3AUnusedPrivateMethod)